package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "js-search": "^1.4.2",
+    "material-design-icons": "^3.0.1",
     "material-ui": "^0.18.1",
```

Comment:
`material-ui` appears to be causing the following error:
`npm WARN inline-style-prefixer@3.0.3 requires a peer of caniuse-api@^1.5.2 but none was installed.`
Would you please add `npm install --save` this to package.json to resolve this dependency error?